raw_data/preprocessed/africa/mauritania_french.txt

```diff
@@ -0,0 +1,251 @@
+
+(71) Les dispositions réglementaires prises en application de la présente loi fixent le (ou les) seuil d'intensités de bruits ou de vibrations à ne pas dépasser et prévoient les systèmes de mesure et les moyens de contrôle à mettre en oeuvre pour assurer le respect des seuils admissibles.
+
+CHAPITRE IV - ODEURS, POUSSIÈRES ET LUMIÈRES INCOMMODANTES
+
+(72) Les odeurs nauséabondes doivent être supprimées dans toute la mesure du possible.
+
+(73) En vue de prévenir et de lutter contre les odeurs et la pollution atmosphérique, des mesures doivent être prises conformément aux dispositions de la présente loi et des textes en vigueur. Ces mesures doivent préciser notamment les caractéristiques des équipements sanitaires individuels et collectifs autorisés, les conditions d'implantation et d'ouverture des décharges publiques ou privées, ainsi que les conditions d'exercice de toute activité susceptible d'émettre des odeurs nauséabondes.
+
+(74) Sont interdites, dans tout établissement, habitation, agglomération, la production de poussière, de fumées épaisses, notamment suies, buées et de façon générale, toutes projections et émanations susceptibles de nuire à la santé et à la commodité du voisinage au delà des seuils prévus par voie réglementaire.
+
+(75) Toute utilisation de sources lumineuses à rayonnements nuisibles sans respect des conditions de protection de la santé et de l'environnement est interdite. Les dispositions réglementaires prises en application de la présente loi préciseront la nature de ces rayonnements.
+
+CHAPITRE V - DÉGRADATIONS DE L'ESTHÉTIQUE ENVIRONNEMENTALE
+
+(76) Au sens de la présente loi, on entend par dégradation de l'esthétique environnementale toutes actions tendant à avilir le milieu sous quelque forme ou quelque degré que ce soient, notamment par: [1] la salissure des murs, parois ou façades des immeubles, édifices, monuments ou autres; [2] l'obstruction, l'occupation abusive, l'encombrement ou l'enlaidissement des voies de circulation et des lieux publics.
+
+(77) Les agents du Ministère chargé de l'Environnement dûment habilités apprécient la réalité et le degré de la dégradation de l'esthétique.
+
+(78) Afin d'assurer la protection et la sauvegarde de l'esthétique du milieu, iI est fait obligation à toute personne physique ou morale concernée: [1] de réaliser des constructions selon les plans cadastraux et dans le respect des règles d'urbanismes; [2] de combattre toutes les pollutions et nuisances découlant aussi bien des activités économiques et sociales que du processus biologique; [3] de déterminer rationnellement les découpages des territoires urbains et ruraux; [4] d'adopter des mesures de protection appropriées s'appliquant aux zones d'habitat ou aux zones affectées aux activités industrielles ou touristiques et aux installations de dépôts pour les déchets et les résidus.
```

Comment:
The "iI" in "iI est fait" looks like an "i" followed by a capital "i" instead of a "l".

Reply:
You are correct. I will amend.